### PR TITLE
feat(eventstream): Add consumer strategy for post process forwarder

### PR DESCRIPTION
**Pull request by lynnagara**
Processing steps for the Arroyo version of the post process forwarder. This is close to equivalent to what the non-Arroyo post process forwarder consumer is doing today. It's not yet being used. The consumer and update to the CLI entrypoint will be added as a follow up.

### CODE REVIEW

File: src/sentry/eventstream/kafka/consumer_strategy.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+def _get_task_kwargs_and_dispatch(message: Message[KafkaPayload]) -> None:
+    task_kwargs = _get_task_kwargs(message)
+    if not task_kwargs:
+        return None
+
+    dispatch_post_process_group_task(**task_kwargs)
+
+
+class DispatchTask(ProcessingStrategy[KafkaPayload]):
```
Comment (lynnagara) [inline src/sentry/eventstream/kafka/consumer_strategy.py:75]: Will eventually be replaced by Arroyo's RunTask when that is available https://github.com/getsentry/arroyo/pull/111

**Review by nikhars — APPROVED** [review]

File: tests/sentry/eventstream/kafka/test_consumer_strategy.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    "project_id": 1,
+                    "primary_hash": "311ee66a5b8e697929804ceb1c456ffe",
+                },
+                {
+                    "is_new": False,
+                    "is_regression": None,
+                    "is_new_group_environment": False,
+                    "skip_consume": False,
+                },
+            ]
```
Comment (fpacifici) [inline tests/sentry/eventstream/kafka/test_consumer_strategy.py:33]: Now that we have more use cases with different schemas (see performance issues) and that, with arroyo, we can test the consumer logic much more thoroughly do you have a plan to improve the test coverage of the forwarder overall ?

**Review by fpacifici — COMMENTED** [review]
Looks good to me, though I think we should improve the test coverage overall in the post process forwarder now that we can run unit tests with arroyo.

File: tests/sentry/eventstream/kafka/test_consumer_strategy.py (hunk 2 above)
Comment (lynnagara) [reply tests/sentry/eventstream/kafka/test_consumer_strategy.py:33]: Honestly not really. The forwarder does not concern itself with schemas at all and has close to no product logic. It just forwards a message of any shape. The underlying dispatch function was not touched at all.

**Pull request merged**